code/datums/martial/wrestling.dm

```diff
@@ -417,7 +417,7 @@
 
 		if (falling == 1)
 			if (prob(33) || D.stat)
-				D.ex_act(EXPLODE_LIGHT)
+				SSexplosions.lowturf += D
```

Comment:
For all of these being replaced with SSexplosions.lowturf += D, I feel like keeping the ex_act proc would be better an have the proc call SSexplosions.lowobj += src depending on what you call it from, and then obviously override it on turfs to add itself to low/med/highturf depending on the severity passed in. Would make porting anything with explosions easier

Reply:
I can do that... just figured accessing a list would be less intensive then calling an entire proc that just does the same thing.